packages/backend/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@clerk/backend",
+  "version": "0.0.1",
+  "license": "MIT",
+  "description": "Clerk Backend SDK - REST Client for Backend API & JWT verification utilities",
+  "types": "./dist/types/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "dev": "tsup --watch",
+    "build": "run-s clean build:runtime build:lib",
+    "build:lib": "tsup --env.NODE_ENV production",
+    "build:tests": "tsc -p tsconfig.test.json",
+    "build:runtime": "mkdir -p dist/runtime && cp -r src/runtime/* dist/runtime",
+    "clean": "rimraf ./dist",
+    "lint": "eslint .",
+    "test": "run-s build:runtime build:tests test:node test:edge-runtime test:cloudflare-miniflare",
+    "test:node": "./tests/node/run.sh",
+    "test:edge-runtime": "./tests/edge-runtime/run.sh",
+    "test:cloudflare-miniflare": "tests/cloudflare-miniflare/run.sh",
+    "test:cloudflare-workerd": "tests/cloudflare-workerd/run.sh"
+  },
+  "dependencies": {
+    "@clerk/types": "^3.15.2-staging.1",
+    "@types/node": "^16.11.55",
+    "node-fetch-native": "^0.1.7",
+    "rfc4648": "^1.5.2",
+    "snakecase-keys": "^5.4.4",
+    "tslib": "^2.3.1"
```

Comment:
❓  Shouldn't we pin down the dependencies versions?

Reply:
We will do it once for all packages.